### ISSUE TITLE
P1 experimental production lens: expose literal loudness, spatial, and timbre changes

### ISSUE THREAD

**Issue by gr-rr** [ListenCloser/listencloser#1179]
Parent research: #1172. Experimental UX owner: #1173. Existing perceptual evidence/history: #455/#468. Projection/focus: #807. Grounded finding consumer: #588.

## Product job

> **How does the sound itself change across this recording?**

This is a cheap, independent second-wave breadth lane. It should deepen produced-music understanding with literal measurements before adding more semantic models.

## OSS-first direction

Prefer transparent maintained primitives:

- existing **librosa** for selected spectral/transient descriptors already compatible with the worker;
- **pyloudnorm** for ITU BS.1770-style loudness / loudness-range measurements where the duration/gating contract is appropriate;
- small transparent NumPy/SciPy channel calculations for mid/side energy, channel balance, and inter-channel correlation when no additional dependency is needed.

Integration level: **1–2**.

Do not adopt Essentia or another broad extractor merely to gain a catalog of descriptors; historical licensing/runtime constraints remain relevant.

## Candidate measurements

Start with the smallest set that creates musician-facing relations, such as:

- integrated or short-term loudness / LRA where method semantics are valid;
- stereo width or side-vs-mid energy;
- inter-channel correlation / balance;
- spectral contrast;
- spectral flatness;
- spectral rolloff / high-frequency redistribution;
- transient/percussive balance;
- bounded dynamic change over trusted or user-selected spans.

Do not compute every available descriptor.

## Product relations

The product may expose method-qualified observations such as:

```text
This passage measures quieter than the preceding span under BS.1770 loudness.
This span has more side-channel energy than the comparison span.
High-frequency spectral energy is redistributed upward here.
Transient/percussive activity increases across this boundary.
```

Prefer within-Work selected-span or trusted-span comparison first. Reuse current shared time/selection/focus and Breakdown admission rules.

## Truth boundaries

- Measurements remain literal and method-qualified.
- Do not map centroid/contrast/flatness/loudness into `bright`, `warm`, `huge`, `exciting`, `wide`, `intense`, or causal production prose unless a later interpretation layer explicitly qualifies it.
- Stereo-dependent evidence must use the original stereo source path; do not silently compute spatial claims after mono downmix.
- Preserve sample-rate/channel/preprocessing and exact source Version provenance.
- Respect #455/#468 stability lessons: descriptor semantics can change with preprocessing and boundary choice.
- A relation score is not importance or confidence.

## First product slice

Prefer one compact **Sound / Production** lane or a few high-value Breakdown observations over a generic descriptor dashboard. A temporal result should have an immediate Hear/Focus action and one proof/provenance disclosure.

Do not create one control/lane per descriptor.

## Product proof

Inspect several materially different lawful Works, including stereo produced music and a control where spatial evidence is unavailable/meaningless. Record whether the surfaced differences are audible/useful and delete descriptors that do not earn product space.

## Canonization later

Only later evaluate broad cross-encode/cross-Work comparability, default thresholds, or semantic labels. First experimental exposure stays within-Work, literal and method-qualified.

## Acceptance

- [ ] a deliberately small literal production/spatial evidence set is implemented with exact source/preprocessing provenance;
- [ ] stereo-specific evidence never silently uses mono inputs;
- [ ] at least two useful within-Work relations can be heard/focused in the product;
- [ ] unsupported measurements fail closed rather than fabricating values;
- [ ] product UI groups the result by musician job rather than descriptor names;
- [ ] several real Works are manually inspected and low-value measurements are withheld;
- [ ] no generic descriptor dashboard, semantic-adjective mapper, or broad evaluator is introduced.

Refs #1172 #1173 #455 #468 #588 #807 #1064.

**Comment by gr-rr** [comment]
## Research follow-on trigger — do not activate yet

Keep #1179 product-first and literal. Its current job remains shipping/inspecting standards-based loudness, stereo/spatial, spectral and transient evidence with no semantic ML layer.

If real product use shows that these literal production relations are useful **and** exposes a concrete residual job such as `what changed between mix/version A and B?`, #1303 may activate one cheap proprietary research probe rather than training a generic production model.

Suggested future probe:

```text
rights-safe source audio
→ deterministic known DSP transforms
   EQ / gain / compression / reverb / stereo width / distortion / panning
→ paired source/modified examples with exact transform labels
→ compare:
   A. literal DSP feature differences
   B. frozen general audio representation
   C. effect-specific representation (Fx-Encoder++ class challenger if runnable/rights-safe)
   D. tiny pairwise relation head only if needed
```

Prefer maintained tooling such as `dasp-pytorch` or equivalent exact DSP primitives; do not implement a DSP framework. Synthetic transform parameters are the ground truth, so no manual labeling is required.

Potential later product consumer:

> Compare two mixes/Versions and explain the **measured/recognized production relations that changed**, with exact A/B audition.

Activation gate: #1179 must first demonstrate that literal Production/Sound evidence earns product space. If it does not, do not create this research program. If simple DSP measurements already recover the useful relation, keep them and do not train a model.

Refs #1303 #1297.
